src/plays/calendar/Calendar.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { getPlayById } from 'meta/play-meta-util';
+
+import PlayHeader from 'common/playlists/PlayHeader';
+import CalendarGrid from './CalendarGrid'
+import { ContextProvider } from './Context';
+
+import './Calendar.scss'
+import ModalContainer from './ModalContainer';
+
+function Calendar(props:any) {
+  // Do not remove the below lines. 
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={play} />
+        <div className="play-details-body">
```

Comment:
One request here,

You can add one unique class name here like,

```html
<div className="play-details-body some-unique-class-name">
```

And use the `some-unique-class-name` in your SASS file to make sure there is a CSS specificity applied as in no chances of conflicts